Project/library/include/HistogramProcesser.h

```diff
@@ -11,6 +12,7 @@ class HistogramProcesser : public Processer
 
 	int histogramHeight[256] = {};
 	cimg_library::CImg<unsigned char> histogramImage;
+	std::vector< std::vector<int> > kernels = { { 0,-1,0, -1,5,-1, 0,-1,0 },  { -1,-1,-1, -1,9,-1, -1,-1,-1 },  { 1,-2,1, -2,5,-2, 1,-2,1 } };
```

Comment:
To nie będzie zmieniane, więc myślę że zamiast vector można użyć po prostu int[3][9]